### PR TITLE
Feat/login sign up

### PR DESCRIPTION
**Pull request by KiveshVirasamy**

### CODE REVIEW

**Review by kphutiyagae — COMMENTED** [review]
Seems okay to me. Approved

**Review by kphutiyagae — APPROVED** [review]

File: src/app/shared/services/user.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+
+export interface user {
+    email?: string,
```
Comment (MicNow27) [inline src/app/shared/services/user.ts:4]: Should email be optional?

File: src/app/landing-page/landing-page.component.html
Comment (MicNow27) [inline src/app/landing-page/landing-page.component.html:1]: Perhaps move all components to components folder. Or however you want to organise code, but ... consistency.

File: src/app/app.module.ts [hunk 2]
```diff
@@ -26,7 +34,10 @@ registerLocaleData(en);
   declarations: [
     AppComponent,
     LoginComponent,
-    SignupComponent
+    SignUpComponent,
+    LandingPageComponent,
+
```
Comment (MicNow27) [inline src/app/app.module.ts:39]: Just remember to format code before final commit.

**Review by MicNow27 — APPROVED** [review]
Looks good.

**Pull request merged**